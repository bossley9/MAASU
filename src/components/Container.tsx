import React, { FC, ReactNode } from 'react'

type Props = {
  children: ReactNode
  className?: string
}

export const Container: FC<Props> = ({ children, className = '' }) => (
  <div className={`container py6 ${className}`}>{children}</div>
)
