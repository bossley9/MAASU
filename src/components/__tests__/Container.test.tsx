import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { Container } from '../Container'

describe('render', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Container>hello</Container>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

it('renders a matching snapshot', () => {
  const test = renderer.create(<Container>hello</Container>)
  expect(test.toJSON()).toMatchSnapshot()
})
