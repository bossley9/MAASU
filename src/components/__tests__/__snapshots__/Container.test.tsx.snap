// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a matching snapshot 1`] = `
<div
  className="container py6 "
>
  hello
</div>
`;
